graphqlapi/src/main/java/io/stargate/graphql/web/resources/schemafirst/SchemaFirstCache.java

```diff
@@ -147,11 +157,11 @@ void init() {
     }
 
     private GraphQL buildGraphql() {
-      // TODO handle errors better - things could still fail e.g. if we're loading a namespace that
-      // was not created through this Stargate instance, and the CQL schema doesn't match anymore.
       ProcessedSchema processedSchema =
-          new SchemaProcessor(authenticationService, authorizationService, dataStoreFactory)
+          new SchemaProcessor(authenticationService, authorizationService, dataStoreFactory, true)
```

Comment:
do we need to have the `isPersisted` information in two places?
I can see that we are keeping it in the `SchemaProcessor` but also in the `CassandraMigrator` when calling `forPersisted()`. Isn't it enough to keep it in one place?

Reply:
`SchemaProcessor` and `CassandraMigrator` are independent classes, so we can't pass the information from one to the other.